add more detail around cross-zone load balancing and NLB security groups for PL setups (#6179)
fishtown-analytics/docs.getdbt.com: website/snippets/_privatelink-cross-zone-load-balancing.md

:::note Cross-Zone Load Balancing

> Enabling cross-zone load balancing for your NLB or Target Group is highly recommended, and may be required for some connections. Cross-zone load balancing may also [improve routing distribution and connection resiliency](https://docs.aws.amazon.com/elasticloadbalancing/latest/userguide/how-elastic-load-balancing-works.html#cross-zone-load-balancing). Note that cross-zone connectivity may incur additional data transfer charges, though this should be minimal for requests from dbt Cloud.

Reviewer: ```suggestion
We highly recommend cross-zone load balancing for your NLB or Target Group; some connections may require it. Cross-zone load balancing may also [improve routing distribution and connection resiliency](https://docs.aws.amazon.com/elasticloadbalancing/latest/userguide/how-elastic-load-balancing-works.html#cross-zone-load-balancing). Note that cross-zone connectivity may incur additional data transfer charges, though this should be minimal for requests from dbt Cloud.
```
